tox.ini

```diff
@@ -14,3 +23,12 @@ deps =
     pytest52: pytest==5.2.*
     pytest53: pytest==5.3.*
     pytest54: pytest==5.4.*
+
+[testenv:linting]
+basepython = python3
+commands = pre-commit run --all-files --show-diff-on-failure {posargs:}
+commands_pre =
+    pip install pre-commit
```

Comment:
As I understand tox, this should already be installed, as `pre-commit` is listed in `deps`.

Reply:
it should, you're right. We can remove the entire `commands_pre` section actually